css/properties/background-image.json

```diff
@@ -176,12 +176,17 @@
               "edge": "mirror",
               "firefox": [
                 {
-                  "version_added": "88",
-                  "notes": "Before Firefox 89, the <code>type()</code> function is not supported as an argument to <code>image-set()</code>."
+                  "version_added": "89"
                 },
                 {
                   "prefix": "-webkit-",
                   "version_added": "90"
+                },
+                {
+                  "version_added": "88",
+                  "version_removed": "89",
+                  "partial_implementation": true,
+                  "notes": "The <code>type()</code> function is not supported as an argument to <code>image-set()</code>."
```

Comment:
These are out of order @foolip,

Can you fix this in a follow up?

Reply:
see : https://github.com/mdn/browser-compat-data/issues/23012

Reply:
Unfortunately this can't be fixed without changing the lint, but I've commented on https://github.com/mdn/browser-compat-data/issues/23012 and asked for more feedback.